lib/TH/vector/AVX.h

```diff
@@ -2,22 +2,23 @@
 #define TH_AVX_H
 
 #include <stddef.h>
+#include "THGeneral.h"
```

Comment:
AVX.h is included after THGeneral.h in file ./lib/TH/THVector.c. So there is no need to include THGeneral.h here.